stl/inc/xstring

```diff
@@ -1306,7 +1398,8 @@ public:
         *this  = _Tmp;
     }
 
-    constexpr size_type copy(_Out_writes_(_Count) _Elem* const _Ptr, size_type _Count, const size_type _Off = 0) const {

```

Comment:
Hmmmmm - this was previously `constexpr` because we speculatively implemented the proposed resolution of LWG-2780 (and on 3865). LWG-2780 was resolved by application of WG21-P1032. So while this is a righteous change, I'm concerned about breaking users who took advantage of this previously non-portable behavior in C++17 mode. I think I'd rather keep vanilla `constexpr` on these functions, and continue to have C++17 be not-quite conforming here.

Opinions?

Reply:
*gasps* non conformance *gasps*

Im fine with it but I guess that is not my desicion ;)

Reply:
I highly doubt there are any such users, because the traits `copy` wasn't `constexpr`, so this would only have been possible with a custom traits class.

Reply:
As I mentioned in the other comment, I'd like to resolve this in favor of the righteous, theoretically-source-breaking but in practice nobody will encounter it, change. Please meow if you are concerned.